cmd/kubeadm/app/constants/constants.go

```diff
@@ -138,6 +139,9 @@ var (
 	// DefaultTokenUsages specifies the default functions a token will get
 	DefaultTokenUsages = []string{"signing", "authentication"}
 
+	// DefaultTokenGroups specifies the default groups a token will get
+	DefaultTokenGroups = []string{bootstrapapi.BootstrapDefaultGroup}
```

Comment:
I ended up not needing the import. I think we should discuss the `NodeBootstrapTokenAuthGroup` change in a second PR because that potentially breaks some upgraded clusters if I'm understanding correctly.